shared/src/main/scala/coop/rchain/store/LmdbStoreManager.scala

```diff
@@ -49,6 +48,7 @@ private final case class LmdbStoreManagerImpl[F[_]: Concurrent: Log](
   ) {
     override def toString() = s"DbState(status: $status, inProgress: $inProgress)"
   }
+  implicit private val logSource: LogSource = LogSource(this.getClass)
```

Comment:
This is not necessary because we have macro defined. I added some info as the first item in the list on the issue #3001.

Reply:
Interesting. I didn't put this one before and it didn't compile. But it works now.

Reply:
https://github.com/rchain/rchain/runs/1686812145?check_suite_focus=true if i remove this line, compile error would show.

Reply:
It seems that macros are not supported in the same project.
> The current macro implementation in the compiler requires that macro implementations be compiled before they are used. The solution is typically to put the macros in a subproject or in their own configuration.
https://www.scala-sbt.org/1.x/docs/Macro-Projects.html

I've updated #3001 with this info so we can fix this later as part of cleanup.